client/src/util/finiteExtent.ts

```diff
@@ -6,8 +6,11 @@ If undefined or empty array, or array contains only non-finite numbers,
 will return [undefined, undefined]
 */
 
-// eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types, @typescript-eslint/no-explicit-any -- - FIXME: disabled temporarily on migrate to TS.
-function finiteExtent(tarr: any) {
+import type { TypedArray } from "../common/types/entities";
+
+function finiteExtent(
+  tarr: TypedArray
+): [number | undefined, number | undefined] {
```

Comment:
This signature is wrong as I read the code.  Return type should be:  `[number, number] | [undefined, undefined]`

I can't see a code path that lets you have one defined and the other undefined.